tldraw/tldraw
# fix(ungrouping): prevent deleting locked shape when ungroup

closes https://github.com/tldraw/tldraw/issues/2045

### Change Type


[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Test Plan

1. Add a step-by-step description of how to test your PR here.
- create two shapes
- group them
- make one of the shape locked
- ungroup (the locked shape should not be removed)

### Release Notes
Prevent deleting locked shape when ungrouping
